### PR TITLE
feat(evm): adding call contract with gas estimation

### PR DESCRIPTION
**Pull request by re1ro**
* [x] EVM: `CallContractGasEstimation`
* [x] Depends on GMP-SDK `5.7.0`

### CODE REVIEW

**Review by npty — APPROVED** [review]

**Comment by npty** [comment]
@re1ro I've published `@axelar-network/axelar-local-dev@2.3.2` to bump the dependencies version. here's [the related pr](https://github.com/axelarnetwork/axelar-local-dev/pull/109). could you pls update and try running tests again?

**Comment by re1ro** [comment]
@npty thank you for that! Everything passes on local now. 
But as you see in CI we have a problem with `fsevents` package that makes it fail

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "license": "ISC",
     "dependencies": {
         "@axelar-network/axelar-chains-config": "^0.1.0",
```
Comment (milapsheth) [inline package.json:24]:
```suggestion
        "@axelar-network/axelar-chains-config": "^1.2.0",
```

File: package.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
...
-        "@axelar-network/axelar-gmp-sdk-solidity": "5.6.2",
-        "@axelar-network/axelar-local-dev": "^2.3.0",
+        "@axelar-network/axelar-gmp-sdk-solidity": "5.7.0",
```
Comment (milapsheth) [inline package.json:25]:
```suggestion
        "@axelar-network/axelar-gmp-sdk-solidity": "5.8.0",
```
can bump again

File: examples/evm/call-contract-gas-estimation/CallContractGasEstimation.sol [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+ * @notice Send a message from chain A to chain B and stores gmp message
+ */
+contract CallContractGasEstimation is AxelarExecutable {
+    string public message;
+    string public sourceChain;
+    string public sourceAddress;
+    IAxelarGasService public immutable gasService;
+    uint256 public constant GAS_LIMIT = 200000;
+
+    event Executed(string _from, string _message);
```
Comment (milapsheth) [inline examples/evm/call-contract-gas-estimation/CallContractGasEstimation.sol:20]: add a getter to estimate gas on this contract for composability? It'll internally call `gasService.estimateGas` with the right gas limit set. Calling contracts and the example script can use that instead

Comment (re1ro) [reply examples/evm/call-contract-gas-estimation/CallContractGasEstimation.sol:20]: Implemented!

**Review by milapsheth — APPROVED** [review]

**Pull request merged**